Runtime/Scripts/MobiledgeXLogger.cs

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2018-2021 MobiledgeX, Inc. All rights and licenses reserved.
+ * MobiledgeX, Inc. 156 2nd Street #408, San Francisco, CA 94105
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using UnityEngine;
+
+namespace MobiledgeX
+{
+    public class MobiledgeXLogger
+    {
+        public enum LogType { All, ErrorsAndWarnings, ErrorsOnly}
+        public static LogType logType;
+
+        public static void PrintWarning(string message)
```

Comment:
Although debatable, I still like "Tag" added to Loggers. It allows more creative filtering. Of course, one could also make it part of the original message, if less structured. There's obviously something like message builder if max flexibility is needed.

Also, why "Print"? These things do stream into actual logs in Unity.